moped-editor/src/components/GridTable/Filters.js

```diff
@@ -274,19 +255,17 @@ const Filters = ({
   /**
    * Reset search box and advanced filters
    */
-  const handleResetFilters = useCallback(() => {
```

Comment:
i dont think this needs a `useCallback` anymore, previous this function was called within a `useEffect` but now its not